libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackArrivalFragment.kt

```diff
@@ -0,0 +1,163 @@
+package com.mapbox.navigation.ui.feedback
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MotionEvent
+import android.view.View
+import android.view.View.GONE
+import android.view.ViewGroup
+import android.view.inputmethod.InputMethodManager
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetBehavior.BottomSheetCallback
+import com.google.android.material.bottomsheet.BottomSheetBehavior.from
+import com.google.android.material.snackbar.Snackbar
+import com.mapbox.navigation.core.telemetry.events.FeedbackEvent
+import com.mapbox.navigation.ui.R
+import kotlinx.android.synthetic.main.mapbox_edit_text_feedback_optional_comment.*
+import kotlinx.android.synthetic.main.mapbox_feedback_arrival_bottom_sheet.feedbackArrivalBottomSheet
+import kotlinx.android.synthetic.main.mapbox_feedback_arrival_fragment.mapbox_feedback_arrival_fragment_parent_view_group
+import kotlinx.android.synthetic.main.mapbox_partial_feedback_arrival_button_bar.feedbackArrivalFinishButton
+import kotlinx.android.synthetic.main.mapbox_partial_feedback_bottom_sheet_top_banner.cancelBtn
+import kotlinx.android.synthetic.main.mapbox_partial_feedback_bottom_sheet_top_banner.feedbackBottomSheetTitleText
+
+/**
+ * This [Fragment] is responsible for showing UI related to providing
+ * information.
+ */
+class FeedbackArrivalFragment : DialogFragment() {
```

Comment:
Does this need to be `public`? Can this be `internal`?

Reply:
yes, it can be internal. Same as `FeedbackDetailsFragment`